openjdk-integ-tests/build.gradle

```diff
@@ -24,12 +24,17 @@ dependencies {
                 project(':conscrypt-testing')
 }
 
+test {
+    include "org/conscrypt/*"
```

Comment:
which tests do you expect to not start with `org/conscrypt/`?

Reply:
This is only hitting ConscryptSuite (it's * rather than **, so it's not recursive).  The point is to make it so only the suite is executed rather than having gradle also try to execute the individual test classes outside the suite.  Changed to explicitly list the suite class instead, so it's obvious what's going on.

Reply:
ah ok that makes it more clear ... thanks.